### ISSUE TITLE
Add sensors for S7 maxv dock consumables 

### ISSUE THREAD

**Issue by martinj27** [humbertogontijo/homeassistant-roborock#463]
## Is your feature request related to a problem? Please describe.
A clear and concise description of what the problem is. 
There is currently no sensors for the water filter or the high speed maintenance brush consumables for the S7 maxv dock

## Describe the solution you'd like

I would like these to be added so that they can be viewed and reset like the other consumables 

## Describe alternatives you've considered

I'm currently am using a separate timer in home assistant but I have to reset both home assistant and the Roborock app separately 

## Vacuum

Roborock S7 MaxV Ultra 

## Mqtt request
<!-- If you are looking for a feature that was introduced with a specific vacuum, chances are the reason we have not implemented it is because we need the mqtt request - and we cannot figure it out as we don't have the vacuum. Look here(https://github.com/humbertogontijo/python-roborock/blob/mhttps://https://github.com/humbertogontijo/python-roborock/blob/main/roborock/typing.pygithub.com/humbertogontijo/python-roborock/blob/main/roborock/typing.pyain/roborock/typing.py) to see what commands we have. If we don't have one that corersponds with the feature you are looking for, chances are, we need your (or annother users' help)

If you feel as if you are technically able to do this, please do, if not, just say here that you don't feel comfortable doing this.

You can either create the connection manually using: https://medium.com/geekculture/capture-iphone-ios-http-traffic-using-wireshark-4af01a4313e6

**Comment by Lash-L** [comment]
I don't think we know the replacement times for the consumables, I think that's why they haven't been added yet

**Comment by martinj27** [comment]
The water filter says once a month and the high-speed maintenance brush says 2 months. I don't think it matters if the robot is running or not for these two

**Comment by Lash-L** [comment]
Hm - that's not quite what we need, are you sure it isn't based on runtime at all? I doubt the high speed maintenance brush needs to be replaced every 2 months if you aren't using it at all

**Comment by lightmaster** [comment]
The high speed brush for my S8 Pro Ultra says to remove and wash it every 2 months, not to replace it.
![Screenshot_20240406_202950_Roborock](https://github.com/humbertogontijo/homeassistant-roborock/assets/5307827/00172301-893b-4752-a676-f86ffa938a63)



The water filter says to remove and clean every month, no mention of regularly replacing it.
![Screenshot_20240406_203002_Roborock](https://github.com/humbertogontijo/homeassistant-roborock/assets/5307827/74ca3380-7dcd-4b18-9289-6677d3e1f018)
